src/hooks/useForm.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+export function useForm(inputs = {}) {
+  const [formState, setFormState] = useState(inputs);
+
+  const handleFormChange = (event) => {
+    const { name, value } = event.target;
+    setFormState((prevState) => {
+      return { ...prevState, [name]: value };
```

Comment:
Clever use of destructuring in order to repurpose this handler for multiple inputs!